transcom/mymove
# Prime update mto actual weight

## Description

Create an endpoint that allows PRIME to update the actual weight of a move task order.

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make db_dev_e2e_populate
make db_test_e2e_populate
make server_run
make client_run
```

1. In your SQL editor of choice, create a new row in `move_task_orders` with `actual_weight` set to none.
2. Navigate tohttp://milmovelocal:3000/swagger-ui/ghc.html#/moveTaskOrder/updateMoveTaskOrderActualWeight and click the `Try it out` button.
3. Set the `moveTaskOrderID` to be the same as the MTO you created and put an arbitrary number for the `actualWeight`.
4. Execute.
5. Verify you get a 200 response and the payload looks correct.

## Code Review Verification Steps

## References

* [Pivotal story]https://www.pivotaltracker.com/story/show/169262252) for this change
